src/Data/Skills/sup_str.lua

```diff
@@ -2794,7 +2796,7 @@ skills["SupportLifetap"] = {
 	support = true,
 	requireSkillTypes = { },
 	addSkillTypes = { SkillType.Duration, },
-	excludeSkillTypes = { SkillType.Blessing, },
```

Comment:
I assume this is just a merge issue

Reply:
Yeah not really sure where that came from. Likely from an old merge.